awx/ui_next/src/components/Schedule/Schedule.jsx

```diff
@@ -108,6 +108,11 @@ function Schedule({ i18n, setBreadcrumb, unifiedJobTemplate }) {
           exact
         />
         {schedule && [
+          <Route
+            key="edit"
+            path={`${pathRoot}schedules/:id/edit`}
+            render={() => <ScheduleEdit schedule={schedule} />}
```

Comment:
Instead of using `<Route render />` props, could use switch the Route to use `<Route children />`? This switch will smooth the transition from React Router v5 to v6, plus the maintainers plan on deprecating the `render` and `component` APIs sometime in the future. 
```
<Route>
  <ScheduleEdit />
</Route>